xampp/README.md

```diff
@@ -0,0 +1,17 @@
+# Installing xampp
+
+This example was chosen because it installs to a location unsupported by rpm-ostree(/opt/)
```

Comment:
Makes sense, though @djoshy and I had a realtime chat about this and I think the core tradeoff here is that this code (a webserver app) is generally something we'd expect to be run as a pod.

On the positive side, it is real world code.  But that's a negative; I don't think we should be encouraging users to run webservers directly on the host.

One strawman idea I came up here is that we could make an entirely synthetic app/package which is a "stand-in" for an e.g. monitoring agent that drops its binaries in `/opt`.

Reply:
makes sense to have more realistic example. I honestly thought that we will get easily real world example that have Open Source code but it turned out hard to fine a good one. This is also a good news as majority have moved away from /opt.